websites/M/movie-web/metadata.json

```
@@ -0,0 +1,80 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "705490775932403795",
+		"name": "jordaar"
+	},
+	"service": "movie-web",
+	"description": {
+		"en": "The place for your favorite movies & shows"
+	},
+	"url": "movie-web.app",
+	"regExp": "(dev.)?(movie-web.app)",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/VyageyF.png",
```

Logo must be `512x512`

```suggestion
	"logo": "https://i.imgur.com/EW8lOTs.png",
```